### ISSUE TITLE
DD3 R9. Add a GEMINI element identical to ISO 19115 MD_Identification.purpose

### ISSUE THREAD

**Issue by PeterParslow** [agiorguk/gemini#43]
This makes it clearer to users what the original purpose in collecting and publishing the data was. Note: ISO 19115 only allows a single purpose which is a free text “summary of the intentions with which the resource(s) was developed”. 
If DD3 (or anyone else!) does establish a register of “purposes” then this would be best encoded with a gmx:Anchor to the purpose in that register.

| Name: Purpose       | |
| --------| ----------- |
| Definition          | Summary of the intentions for which the resource was developed |
| Purpose and meaning | To make it clearer to users what the original purpose in collecting and publishing the data was. |
| Obligation          | Optional |
| Occurrence          | Single |
| Data type           | Character string |
| Domain              | Free text |
| Guidance            |  |
| Comment             | If this element is used, the information should not be repeated in the Abstract. |
| Examples            | Natura 2000 report<br>Marine navigation<br><br>Town planning |

**Corresponding element in other standards** 
| Standard                        | Name                                                                            | Comparison                         |
| ------------------------------- | ------------------------------------------------------------------------------- | ---------------------------------- |
| ISO 19115:2003                  | MD\_Identification.purpose                                                      | Identical                          |
| ISO 19139:2007                  | /gmd:MD\_Metadata/gmd:identificationInfo/gmd:MD\_DataIdentification/gmd:purpose | Identical                          |
| [Schema.org](http://Schema.org) | If the resource was developed for education: educationalPurpose                 | The GEMINI element is more general |

**Encoding guidelines**

| Guidelines | If there is a register of purposes (use cases), then include a reference to that, using gmx:Anchor |
| ---------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| Example    | 
```xml
<gmd:MD\_Metadata> 

  ... 
  <gmd:identificationInfo> 
    <gmd:MD\_DataIdentification> 
      ... 
      <gmd:purpose
        <gco:CharacterString>Town planning</gco:CharacterString> 
      </gmd:purpose> 
      ... 
    </gmd:MD\_DataIdentification> 
  </gmd:identificationInfo> 
  ... 
</gmd:MD\_Metadata>
``` |

**Comment by nmtoken** [comment]
Other example of use of `Purpose` from PURE portal description of models:

 - `Distributed Recharge Model`
 - `Numerical ground water flow model`
 - `Landscape model`
 - `Particle Tracking`
 - `Saturated groundwater flow model`
 - `Simulating 3D variably saturated flow with overland flow`
 - `Volcanic Hazard Model`

**Comment by archaeogeek** [comment]
I agree in general, but I don't think it will be possible to link out to a register of purposes as it's a very broad concept. Happy to see it as an optional element encoded as a `gco:CharacterString` though

**Comment by nmtoken** [comment]
We should definitly have the purpose element.  (I see that GeoDCAT  3.1.0 release is adding it so there must be some demand).

Agree that a general register of purposes may be unlikely, but community profiles could indeed have their own codelists, so think it would be odd to restrict to gco:CharacterString

**Comment by archaeogeek** [comment]
So... allow either codelist or character string?
